docs/blog/newsletter-may-2020.rst

```diff
@@ -0,0 +1,91 @@
+*   `Screenpresso <https://www.screenpresso.com/>`__ (Free)
+*   `LightShot <https://app.prntscr.com/en/index.html>`__ (Free)
+*   `Adobe Illustrator <https://www.adobe.com/products/illustrator.html>`__ (Paid)
+*   `Adobe Photoshop <https://www.adobe.com/products/photoshop.html>`__ (Paid)
+
+Join us in `#doctools <https://app.slack.com/client/T0299N2DL/C4EPE8332>`__ if you have any other tool recommendations!
+
+-----------------------------------------
+Choosing the right level to write a topic
+-----------------------------------------
+
+This month, the `#learn-tech-writing <https://app.slack.com/client/T0299N2DL/C7YJR1N02>`__ book club had some interesing discussion based on *Every Page is Page One’s*, on the topic of topics: how do you decide what should go in one? And what's the right level to pitch it at?
+
+Baker’s premise in the relevant chapters (chapters 12 to 13) is that the reader always arrives at the documentation qualified to read and retain the topic they’re searching about; therefore, writing to personas of varying knowledge levels (novice/intermediate/advanced) is not necessary. He argues that although there are several layers of information  available for subjects, the reader is already set on how much information they want to absorb. This can make it challenging to decide when to cover the big picture and when to cover the details.
+
+The book club members listed overviews and "getting started" guides as big picture topics, and setup/requirement instructions and permission levels as details, along with any learning goals relating to “competency.”
+
+Some writers found using precise, clear titles helps keep them focused on the topic at hand; focusing on the title helps you see that anything that falls outside the title’s scope should be assigned its own article. This is more of a gut  determination than one based on metrics, though. It boils down to figuring out the user’s objective: so knowing the persona you are writing to is important when defining what 
+topics to cover and in what depth. Do you write based on user knowledge level? How do you define what a novice is? Someone who could be new to the software may still be an expert in the industry the software services - it's not exactly fair to describe them as a novice.
+
+This is a good opportunity to think about cross-role colloaboration, too. It can be argued when writing documentation, a technical writer should simply present the information as it is; and when developing learning paths, an instructional designer can help with building the path and deciding on the appropriate learning level.
+
+---------
+Job posts
+---------
+
+* `Technical Writer <https://jobs.writethedocs.org/job/197/technical-writer/>`__
+   Semaphore, remote, full- or part-time
+
+*To apply for these jobs and more, visit the* `Write the Docs job board <https://jobs.writethedocs.org/>`_.
+
+--------------------------
+Community events coming up
```

Comment:
Insert disclaimer about all of these being virtual?

Reply:
done